IBEX_device_generator.py

```diff
@@ -57,11 +61,11 @@ def main():
     parser.add_argument("--name", type=str, help="Name of the device", required=True)
     parser.add_argument("--ticket", type=int, help="Ticket number", required=True)
     parser.add_argument("--device_count", type=int, help="Number of duplicate IOCs to generate", default=2)
-    parser.add_argument("--use_git", action='store_true', help="Currently not working - Use to create relevant "
-                                                               "branches and support repository")
+    parser.add_argument("--use_git", action='store_true', help="Use to create relevant branches and support repository")
```

Comment:
I think we should now make this default to true (by using store_false?) and make the --token argument required as it now works